.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,10 +23,10 @@
 <!-- check the items below that will be completed prior to merge.
      strikethrough any item text that does not apply to this PR. -->
 
-- [ ] :ok_hand: design updates will be Garden Designer approved (add the designer as a reviewer)
+- [ ] :ok_hand: design updates are Garden Designer approved (add the designer as a reviewer)
 - [ ] :globe_with_meridians: demo is up-to-date (`yarn start`)
 - [ ] :arrow_left: renders as expected with reversed (RTL) direction
 - [ ] :metal: renders as expected with Bedrock CSS (`?bedrock`)
-- [ ] :guardsman: includes new unit tests
-- [ ] :wheelchair: tested for [WCAG 2.1](https://www.w3.org/TR/WCAG21) AA compliance
+- [ ] :guardsman: includes new unit tests. Maintain [existing coverage](https://coveralls.io/github/zendeskgarden/react-components?branch=main) (always >= 94%)
+- [ ] :wheelchair: tested for [WCAG 2.1 AA](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa) Accessibility compliance
```

Comment:
```suggestion
- [ ] :wheelchair: tested for [WCAG 2.1 AA](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa) accessibility compliance
```

[nit] re: https://garden.zendesk.com/content/grammar#capitalization

Pro move on the URL. Nice.

Reply:
de-nitted